### PR TITLE
repalce `createCanvasContext`(2.9.0 deprecated)

### PR DESCRIPTION
**Pull request by xjh093**
repalce `createCanvasContext`(2.9.0 deprecated) with `createSelectorQuery`
fix Issues [35](https://github.com/chanind/hanzi-writer-miniprogram/issues/35) [36](https://github.com/chanind/hanzi-writer-miniprogram/issues/36) [51](https://github.com/chanind/hanzi-writer-miniprogram/issues/51) [52](https://github.com/chanind/hanzi-writer-miniprogram/issues/52)

### CODE REVIEW

File: src/RenderTarget.js [hunk 1]
```diff
@@ -30,8 +30,23 @@ class RenderTarget {
   constructor(view) {
     this.view = view;
     this.eventEmitter = new EventEmitter();
-    this.ctx = polyfillCanvasCtx(wx.createCanvasContext('writer-canvas', view));
-    this.canvas = this.view.selectComponent('#writer-canvas');
+    //this.ctx = polyfillCanvasCtx(wx.createCanvasContext('writer-canvas', view));
+    //this.canvas = this.view.selectComponent('#writer-canvas');
```
Comment (chanind) [inline src/RenderTarget.js:34]: Thanks for fixing this! Could delete these commented lines?

File: src/RenderTarget.js [hunk 2]
```diff
@@ -30,8 +30,23 @@ class RenderTarget {
...
+    
+    // repalce `createCanvasContext`(2.9.0 deprecated) with `createSelectorQuery`
```
Comment (chanind) [inline src/RenderTarget.js:36]: nit: replace

**Review by chanind — CHANGES_REQUESTED** [review]
Just some minor nits about comments, but otherwise looks good!

File: src/RenderTarget.js (hunk 2 above)
Comment (xjh093) [reply src/RenderTarget.js]: I have deleted the `replace` commented line.

Comment (xjh093) [reply src/RenderTarget.js]: should I also delete these lines:
```
    //this.ctx = polyfillCanvasCtx(wx.createCanvasContext('writer-canvas', view));
    //this.canvas = this.view.selectComponent('#writer-canvas');
```

**Review by chanind — APPROVED** [review]

**Comment by chanind** [comment]
I merged this into the beta branch so it can be tested out just to make sure it works for everyone. You can install it with `npm install hanzi-writer-miniprogram@beta`. If you can try that out, could you let me know if it works as expected, and I'll merge it into master?

**Comment by xjh093** [comment]
I have test for wechat lib 2.8.3，2.9.5，2.10.4，2.19.6，2.21.0，it display well~
image

**Pull request merged**